docs/quant/2_access_quant_1_dataset.md

```diff
@@ -0,0 +1,82 @@
+
+- **Owner** column: 'Owner' means the submitter who designed the experiments, set up this dataset, and submitted files, including raw.files, for the first time. The column of 'Owner' shows a summary of what the owner provided.
+
+- **Reanalysis** column: First, it shows how many reanalyses are included in this dataset. The column of 'Reanalyses' is the summary of all reanalyses in this dataset. For this example, 51 reanalyses are available in this dataset. The column of '51 Reanalyses' is the summary of 51 reanalyses.
+
+- **N/A**: Not available. The information corresponding to the summary is not submitted. For example, MassIVE.quant can't count the number of Conditions, Biological Replicates, or Technical Replicates if the owner didn't provide any files in `Metadata` category. Then, 'N/A' will be shown in the 'Owner' column for 'Experimental Design'. 
+
+- **i** in the blue circle: Detailed information for each row.
+
+
+1. **Experimental Design**: The summary of detected experimental design from `Metadata` category.
+
+    - **Condition**: Number of distinct conditions across all analyses (original submission and reanalyses) associated with this dataset. Distinct condition labels are counted across all files submitted in the `Metadata` category having a `Condition` column in this dataset. For this dataset, the owner didn't submit any file in the `Metadata` category ('N/A' shown). However, reanalyses submitted this information. There are four conditions across all files in the `Metadata` category in this dataset's reanalyses.
+
+    - **Biological Replicates**: Number of distinct biological replicates across all analyses (original submission and reanalyses) associated with this dataset. Distinct replicate labels are counted across all files submitted in the `Metadata` category having a `BioReplicate` or `Replicate` column in this dataset. For this dataset, the owner didn't submit any file in the `Metadata` category ('N/A' shown). However, reanalyses submitted this information. Four unique IDs for biological replicates are available across all 'annotation.csv' files in the `Metadata` category in this dataset's reanalyses.
+
+    - **Technical Replicates**: Number of distinct technical replicates across all analyses (original submission and reanalyses) associated with this dataset. The technical replicate count is defined as the maximum number of times any one distinct combination of condition and biological replicate was analyzed across all files submitted in the `Metadata` category. In the case of fractionated experiments, only the first fraction is considered. For this dataset, the owner didn't submit any file in the `Metadata` category ('N/A' shown). However, reanalyses submitted this information. The Maximum three technical replicates for the corresponding biological replicate are available across all reanalyses in this dataset.
+
+2. **Identification Result**: The summary of the result for identification searching from `xx` category.
```

Comment:
Maybe instead: The summary of the identifications from searching the dataset.

Also, what does 'xx' refer to here?